crates/autopilot/src/infra/persistence/mod.rs

```diff
@@ -140,6 +147,37 @@ impl Persistence {
 
         ex.commit().await.context("commit")
     }
+
+    /// Get normalized token prices.
+    pub async fn normalized_auction_prices(
```

Comment:
Would it be more ergonomic to provide a way to recover the entire auction struct as we sent it to the drivers in the first place?

Reply:
We currently have no needs for that. But we will need fee policy also. So maybe it really makes sense. Will think about it.

Reply:
I think we can do this in a follow up PR where I want to add a check if a settlement has the promised score. Too much for this PR.